modules/widgets/google-translate.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/*
+Plugin Name: Google Translate Widget for WordPress.com
+Plugin URI: http://automattic.com
+Description: Add a widget for automatic translation
+Author: Artur Piszek
+Version: 0.1
+Author URI: http://automattic.com
+Text Domain: jetpack
+*/
+
+class Google_Translate_Widget extends WP_Widget {
+	static $instance = null;
+
+	function __construct() {
+		parent::__construct(
+			'google_translate_widget',
+			__( 'Google Translate (Jetpack)', 'jetpack' ), 
```

Comment:
It's best to use the `jetpack_widget_name` filter here to add `(Jetpack)` to the end of the widget name.
https://github.com/Automattic/jetpack/blob/4.3.2/modules/widgets/facebook-likebox.php/#L38